### ISSUE TITLE
mobilenetv2 poor accuracy

### ISSUE THREAD

**Issue by kraoa** [keras-team/keras#13293]
I imported mobilenetv2 but the accuracy is incredibly low (close to 0% correct)

My code is as below

from keras.applications.mobilenet_v2 import MobileNetV2
model1 = MobileNetV2(weights='imagenet')

import cv2
x = cv2.imread('image_input.jpg')
x = cv2.resize(x,(224,224))
y = model1.predict(x[np.newaxis,:,:,:])

Wheras if I import VGG (or resnet)
from keras.applications.vgg16 import VGG16
model2 = MobileNetV2(weights='imagenet')
y = model2.predict(x[np.newaxis,:,:,:])

VGG has a much much better accuracy

Has keras trained mobilenetv2 properly.

**Comment by jokandre** [comment]
Are you normalizing the images properly?
I have noticed that pre-trained models tend to have a preprocess_input function. 
From the documentation:
from keras.applications.resnet50 import preprocess_input

MobileNetV2 also has one: [see here](https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v2.py)

**Comment by kraoa** [comment]
Yes input needs to be in the [0,1] range as opposed to [0,255] that it was before. Once it is in [0,1], the accuracy improves significantly.

**Comment by jokandre** [comment]
That's the scale. The preprocess_input() applies scaling and normalization using standard deviation and mean. 
From my experience the images tend to become darker.
